internal/graphql/resolver/addons.go

```diff
@@ -6,163 +6,114 @@ import (
 	"strings"
 
 	"github.com/layer5io/meshery/internal/graphql/model"
+	"github.com/layer5io/meshkit/utils"
```

Comment:
I didn't notice this filename previously. Calling Prometheus and Grafana "add-ons" is a carry-over from Istio and not entirely how Meshery views such integrations. Meshery's use of either the of the two of these spans well past Istio - past any service mesh for that matter.